op-conductor/README.md

```diff
@@ -53,9 +53,9 @@ Below is a high level summary of how each failure scenario is handled:
 
 | Failure Scenario | Category | Scenario and Solution | Notes |
 | ---------------- | -------- | --------------------- | ----- |
-| 1 sequencer down, stopped producing blocks | sequencer failure | Conductor will detect sequencer failure and start to transfer leadership to another node, which will start sequencing instead | Cannot tolerate 2 node failures, at that time, would need to manually start sequencing on the remaining sequencer |
```

Comment:
Seems like lowercase “sequencer” is used in other places so let’s not make this change for now.